arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -39,5 +40,19 @@ public async Task<GetGraphsResponse> GetGraphs()
                 throw await GetApiErrorException(response);
             }
         }
+
+        public async Task<DeleteGraphResponse> DeleteGraphAsync(string graphName, DeleteGraphBody body)
+        {
+            StringContent _body = GetStringContent(body, true, true);
```

Comment:
It's a bit odd to see a variable starting with underscore because this convention is for private class member variable. Maybe stick with `content` like in other methods?